### PR TITLE
Hotfix/restore cdb export

### PR DESCRIPTION
**Pull request by LTA-Thinking**
## Description
Hotfix to revert Cosmos export to the legacy system.

## Related issues
The new system was causing issues with runaway storage size increase and locking up the service with high RU consumption in certain edge cases.

## Testing
Existing tests from the legacy system.

## FHIR Team Checklist
- **Update the title** of the PR to be succinct and less than 65 characters
- **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
- Tag the PR with the type of update: **Bug**, **Build**, **Dependencies**, **Enhancement**, **New-Feature** or **Documentation**
- Tag the PR with **Open source**, **Azure API for FHIR** (CosmosDB or common code) or **Azure Healthcare APIs** (SQL or common code) to specify where this change is intended to be released.
- [x] CI is green before merge [Build Status](https://microsofthealthoss.visualstudio.com/FhirServer/_build/latest?definitionId=27&branchName=main) 
- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)

### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
Patch|Skip|Feature|Breaking (reason)

### CODE REVIEW

File: src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/54.diff.sql [hunk 1]
```diff
@@ -302,15 +302,15 @@ ALTER PROCEDURE dbo.MergeResources
     ,@TokenTexts dbo.TokenTextList READONLY
     ,@StringSearchParams dbo.StringSearchParamList READONLY
     ,@UriSearchParams dbo.UriSearchParamList READONLY
-    ,@NumberSearchParams dbo.Tmp_NumberSearchParamList READONLY
-    ,@QuantitySearchParams dbo.Tmp_QuantitySearchParamList READONLY
+    ,@NumberSearchParams dbo.NumberSearchParamList READONLY
+    ,@QuantitySearchParams dbo.QuantitySearchParamList READONLY
```
Comment (brendankowitz) [inline src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/54.diff.sql:306]: What is this change?

**Review by brendankowitz — APPROVED** [review]
